api/routes_rtfsp.go

```diff
@@ -561,12 +561,8 @@ func (api *API) createHostedIPFSNetworkEntryInDatabase(c *gin.Context) {
 	}
 	logger.WithField("db_id", network.ID).Info("database entry created")
 
-	// add network to users
-	if err := api.um.AddIPFSNetworkForUser(username, networkName); err != nil {
-		api.LogError(err, eh.NetworkCreationError)(c)
-		return
-	}
 	logger.WithField("user", username).Info("network added to user")
+	// if there is only 1 user, this next part is covered by the previous logic
 	if len(users) > 0 {
```

Comment:
`len(users)` should always be `> 0`: https://github.com/RTradeLtd/Temporal/blob/0717632842fbafa485044e7eb71232b4be01b5c9/api/routes_rtfsp.go#L548

also I dont think this comment is valid anymore

the log on 564 isnt needed anymore either i think

Reply:
Good catch